base/batch/cronjobs/okd-coreos-all-4.13-create-daily.yaml

```diff
@@ -23,6 +23,7 @@ spec:
                 {
                   "params": {
                     "version": "${OKD_VERSION}",
+                    "release-stream": "${RELEASE_STREAM}",
```

Comment:
Do we plan to also add a `branch` parameter to pass along, so that we can build from `release-4.13` instead of master (default value)

Reply:
Good catch! I wasn't able to test the cronjobs yet. We should add that now.

Reply:
done